design.md

```diff
@@ -2,12 +2,32 @@
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
 
+We are making this design for programmers more comfortable with imperative programming (i.e. while loops).
+
+We assume that the users understand while-loops and some level of imperative programming. It would be helpful to understand the cardinal directions (NEWS) and boolean `&&`. 
+
+
 ## Why did you choose this design, i.e., why did you think it would be a good idea for users to express the maze-searching computation using this syntax?
 
+We want to express picobot in different types of programming. So instead of using states, we use imperative while-loops. This can allow users to program the same functionality using different methods and styles. Just like how we learned Scala by re-implementing what we know, users can learn imperative AND state-based programming by using Dodds' syntax and our syntax.
+
+
 ## What behaviors are easier to express in your design than in Picobot’s original design?  If there are no such behaviors, why not?
 
+It is easier to express repeated actions without using states. It is also easier to express things in a chronological fashion (whereas states can be arbitrary).
```

Comment:
Agreed -- this was also one benefit we saw in using a while loop-like structure.